release/proto/openapiv2/rpc/api/v1/api.swagger.json

```diff
@@ -127,6 +127,28 @@
     }
   },
   "definitions": {
```

Comment:
I don't think we need the openAPI definition of the poet API. It is usually only used to host a Swagger UI: https://petstore.swagger.io/ so we could remove it from `buf.gen.yaml` 🙂

Reply:
Opened https://github.com/spacemeshos/poet/issues/434 to do it.